File: commands/verify.js
const { Command, Middleware } = require('yamdbf');
const { resolve, expect } = Middleware;
const utils = require('../utils');

module.exports = class extends Command {
	constructor() {
		super({
			name: 'verify',
			desc: 'Verify ownership of a forum profile',
			usage: '<prefix>verify <code>',
		});

		this.use(resolve('code: String'));
		this.use(expect('code: String'));
	}

	action(message, args) {
		// Check code from database
        utils.post('verify.php', { 
            discord_id: message.author.id,
            code: args[0],
        }, body => {
            if (body == 'success') {
                // Respond with confirmation
                message.reply('Hurray! You have successfully linked your Discord Account with your Forum Profile.');
            } else {
                // Respond with error
                console.log(body);
                message.reply('Something went wrong. Please check the code you entered and try again.');
            }
        });
	}
};

File: commands/verify.py
from commands.base_command  import BaseCommand
import requests
import settings
from utils                  import get_base_url, get_emoji

class Verify(BaseCommand):
    def __init__(self):
        description = "Verify ownership of a forum profile"
        params = ['code']
        super().__init__(description, params)

    async def handle(self, params, message, client):
        url = get_base_url() + 'verify.php'
        data = {
            'discord_id': message.author.id,
            'code': params[0],
        }
        r = requests.post(url, data)
        
        if r.text == 'success':
            msg = 'Hurray! You have successfully linked your Discord Account with your Forum Profile.'
        else:
            print(r.text)
            msg = 'Something went wrong. Please check the code you entered and try again.'
        
        await client.send_message(message.channel, msg)

